### PR TITLE
Fix/enhancements

### PR DESCRIPTION
**Pull request by GoOz**
Several enhancements & fixes required by @kelukelu:
- [x] fix: Mobile Download app button wasn't displayer at precisely 1023px
- [x] feat: better navigation in viewer component with hover state
- [x] fix: Used shared styles from UI in selection bar
- [x] fix: toolbar on timeline showed wrong actions
- [x] feat: upgrade loading component so it can change its color
- [x] feat: update layout so scrolling fits well with no extra padding
- [x] fix: albumPhotos layout
- [x] Fix: Timeline justified layout (ping especially @goldoraf & @gregorylegarec on this one -> https://github.com/GoOz/cozy-photos-v3/commit/e984c8963a264a017f8b35f2d92f13bec4c16e7e )
- [x] fix: add2album modal style
- [x] fix: edit album's input was too big
- [x] fix: error component style

⚠️ wait for a release of Cozy-UI

### CODE REVIEW

**Comment by GoOz** [comment]
[(0,5j + 0,5j) Cosmétique : pairing avec Jean-Luc](https://trello.com/c/8yedNmqO/341-05j-05j-cosm%C3%A9tique-pairing-avec-jean-luc)

File: src/containers/SelectionBar.jsx [hunk 1]
```diff
@@ -24,15 +24,15 @@ const SelectionBar = ({ t, selected, selectedCount, album, onHide, onAddToAlbum,
     <span className={styles['coz-selectionbar-separator']} />
     <button
       disabled={selectedCount === 0}
-      className={styles['pho-action-album-add']}
+      className={styles['coz-action-album-add']}
```
Comment (goldoraf) [inline src/containers/SelectionBar.jsx]: Is it wise to move albums-related styles to cozy-ui? These styles look app-specific to me.

File: src/ducks/timeline/components/Toolbar.jsx [hunk 2]
```diff
@@ -37,11 +35,12 @@ export const Toolbar = ({ t, disabled = false, uploadPhotos, deleteAlbum, select
           disabled={disabled}
           label={t('Toolbar.menu.photo_upload')}
           type='menu-item'
+          className='coz-mobile'
         />
       </Item>
-      <hr />
+      <hr className='coz-mobile' />
```
Comment (goldoraf) [inline src/ducks/timeline/components/Toolbar.jsx]: Not sure about the naming here...

Comment (GoOz) [inline src/ducks/timeline/components/Toolbar.jsx]: What do you mean? That's an old utility class actually.

File: src/containers/SelectionBar.jsx (hunk 1 above)
Comment (GoOz) [inline src/containers/SelectionBar.jsx]: true. but it was already in Cozy-UI, so I made the app use it instead of rewriting the style. :/

**Pull request merged**